.github/workflows/continuous-integration.yml

```diff
@@ -43,4 +43,6 @@ jobs:
       with:
         name: package
         path: package/*.nupkg
-        if-no-files-found: error   
\ No newline at end of file
+        if-no-files-found: error
+    # - name: Publish to nuget
```

Comment:
These steps are being run on every CI build including PRs which would mean publishing a nuget.

Reply:
It's actually commented out, but I'll be raising another PR to 'simulate' that and remove the commented job.

Reply:
Hello @bingenito I've updated the PR.